paymentsheet/src/main/java/com/stripe/android/paymentsheet/IntentConfirmationHandler.kt

```diff
@@ -492,18 +453,25 @@ internal class IntentConfirmationHandler(
     }
 
     private fun onPaymentResult(result: InternalPaymentResult) {
```

Comment:
Right now, we still have to manage the launcher in the handler. In a follow-up PR, there will be a `Mediator` typed to the definition file at hand that will manage this.